test/revert-cleanup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "Revert Ganache to Pre-Setup"
+cd dex-contracts
+truffle exec scripts/ganache/revertSnapshot.js $1
+
+echo "Dropping Deposits"
+mongo dfusion2 --eval "db.deposits.drop()" | grep -w true
```

Comment:
Should these only drop deposits up to a certain index? Otherwise the snapshot and database become out of sync.

Reply:
Not sure I understand. All the unit tests have been updated to include any necessary deposits for themselves. This script basically resets the DB to its "contract deployment" state. I suppose it could be more refined, the the `drop` should be general enough to handle all test cases.

Reply:
this is removed - some of the lost, but useful commands have been moved into the wiki.